tests/test_transforms/test_inference/test_inverse_transform.py

```diff
@@ -2589,6 +2595,7 @@ def _test_inverse_transform_future_with_target(
                 "regular_ts",
                 {},
             ),
+            # (FourierDecomposeTransform(in_column="target", k=5, residuals=True), "regular_ts", {}),
```

Comment:
Why is this commented?

Reply:
In such cases dataset for transform has only timestamps from future. So error will be raised and test would never pass.